.github/run_translation_verification.sh

```diff
@@ -0,0 +1,24 @@
+
+#!/bin/bash
+
+set -e
+
+# Verifiy that the messages.pot file in the repo is up to date
+# If there are any git diffs after translations are extracted, the checked-in file is out of date
+npm run extract-translations
+
+git diff --exit-code translations/messages.pot > /dev/null # send stdout to /dev/null to reduce clutter in the CI output
```

Comment:
I think we can use something like the $GITHUB_REF env var to always run the messages.pot verification, but only run the [lang].po check for master and release branches

Reply:
Ah actually GITHUB_BASE_REF might be better

Reply:
In the run_browserstack_acceptance.sh script we use $GITHUB_BRANCH. Does that work?

Reply:
I thiink that might be the branch being merged into the base branch? Not sure though

Reply:
updated!

Reply:
oh wait I did not realize we were already using it for filtering lol, think I was looking at the version on master